.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+### Describe the change
+
+Explanation of what this PR does
+
+### Steps to test the PR
+
+Recommendations for how to test this PR
+
+### Issue reference
+
+If this PR is related to a github issue, please link it here (e.g., Closes #100)
```

Comment:
```suggestion
If this PR is related to a github issue, please link it here (https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
```

I think it is better to refer to github link that shows how to link an issue in a PR. With the example you have the risk to link multiple PRs to issue 100 if the comment is not removed.